talk/owt/sdk/base/peerconnectiondependencyfactory.h

```diff
@@ -73,9 +73,9 @@ class PeerConnectionDependencyFactory : public rtc::RefCountInterface {
   CreatePeerConnectionOnCurrentThread(
       const webrtc::PeerConnectionInterface::RTCConfiguration& config,
       webrtc::PeerConnectionObserver* observer);
-#if defined(WEBRTC_WIN) || defined(WEBRTC_LINUX)
-  rtc::scoped_refptr<webrtc::AudioDeviceModule> CreateCustomizedAudioDeviceModuleOnCurrentThread();
-#endif
+  rtc::scoped_refptr<webrtc::AudioDeviceModule>
```

Comment:
why will you need this for iOS?

Reply:
iOS supports custom audio input as well.